interface/main/tabs/menu/menu_json.php

```diff
@@ -1,41 +1,54 @@
 <?php
 /**
- * Copyright (C) 2016 Kevin Yeh <kevin.y@integralemr.com>
- *
- * LICENSE: This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 3
- * of the License, or (at your option) any later version.
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- * You should have received a copy of the GNU General Public License
- * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ * Menu loader.
  *
  * @package OpenEMR
- * @author  Kevin Yeh <kevin.y@integralemr.com>
- * @author Robert Down <robertdown@live.com>
  * @link    http://www.open-emr.org
+ * @author  Kevin Yeh <kevin.y@integralemr.com>
+ * @author  Robert Down <robertdown@live.com>
+ * @author  Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2016 Kevin Yeh <kevin.y@integralemr.com>
+ * @copyright Copyright (c) 2017 Brady Miller <brady.g.miller@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
-
 require_once("menu_updates.php");
 require_once("menu_db.php");
 
 $menu_parsed=load_menu("default");
 if(count($menu_parsed)==0)
 {
-    $menu_parsed = json_decode(file_get_contents(dirname(__FILE__) . "/menu_data.json"));
-    if (!$menu_parsed) die("\nJSON ERROR: " . json_last_error()); // debugging
+    // Use a json file to build menu rather than database
+    // Note this is currently the standard mechanism
+
+    // Collect the selected menu by user
+    $mainMenuRole = "standard";
+    if (!empty($_SESSION['authUser'])) {
+        $sqlResult = sqlQuery("SELECT `main_menu_role` FROM `users` WHERE `username` = ?", array($_SESSION['authUser']));
+        if (!empty($sqlResult['main_menu_role'])) {
+            $mainMenuRole = $sqlResult['main_menu_role'];
+        }
+    }
```

Comment:
@MatthewVita ,
I am planning to migrate above code into services/UserService.php . I just wanted to make sure that would be in scope of the user service.